modules/flashMessage.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-undef */
+/* eslint-disable linebreak-style */
+export default function snaker(type, mes, time) {
+  const para = document.createElement('p');
+  para.classList.add('snakbar');
+  para.innerHTML = `${mes} `;
+  if (type === 'error') {
+    para.classList.add('error');
+  } else if (type === 'success') {
+    para.classList.add('success');
+  } else if (type === 'remove') {
+    para.classList.add('remove');
```

Comment:
Kindly use arrow functions instead of common functions. Arrow functions are a new and faster way to write functions introduced in ES6. :pray: :pray: